deliver/lib/deliver/options.rb

```diff
@@ -205,6 +205,20 @@ def self.available_options
                                      is_string: false,
                                      default_value: :warn),
 
+        # Territory availability
+        FastlaneCore::ConfigItem.new(key: :availability_all_territories,
+                                     description: "Set availability to all territories",
+                                     is_string: false,
+                                     default_value: false),
+        FastlaneCore::ConfigItem.new(key: :availability_territories,
```

Comment:
Does this option and the one below confilct with `availability_all_territories`?

Reply:
Indeed it does, this will take preference over the lists, a validation function was added to the Availability class that will warn the user if any conflicts are detected when uploading.

If you have any ideas on how to make this more clear to users i would be happy to make changes.

Reply:
I think you can pass `conflicting_options: [:available_all_territories]` to the ConfigItem initializer?